LICENSE.txt

```diff
@@ -12,7 +12,7 @@ You may be licensed to use source code to create compiled versions not produced
 2. Under a commercial license available from Mattermost, Inc. by contacting commercial@mattermost.com
 
 You are licensed to use the source code in Admin Tools and Configuration Files (templates/, config/, model/,
-webapp/client, webapp/fonts, webapp/i18n, webapp/images and all subdirectories thereof) under the Apache License v2.0.
```

Comment:
Note that we are removing the mention of the `webapp/` directories as they have been moved to their own repository that is entirely under Apache License v2.0.

Reply:
Thanks, noted in PR description.